### PR TITLE
Work on the What's New

### PR DESCRIPTION
**Pull request by micheles**
See https://github.com/gem/oq-engine/blob/whatsnew/doc/whats-new.md

### CODE REVIEW

File: doc/whats-new.md [hunk 1]
```diff
@@ -1,362 +1,367 @@
...
+
+gmfs_file = gmf-data.hdf5
+
+but that involves some limitations. The HDF5 format of the GMFs is meant
+to be stable across engine versions. However, the details of the logic
+tree implementation change at every release. Therefore the approach can only
+work if risk calculations starting from GMFs in HDF5 format are
+restricted to see a single realization, obtained by collecting together
+all realizations. This is equivalent to using the new option
+`collect_rlzs=true`.
```
Comment (mmpagani) [inline doc/whats-new.md:63]: In which context this should be used? When running a risk analysis?

File: doc/whats-new.md [hunk 2]
```diff
@@ -1,362 +1,367 @@
...
+Moreover, it is possible to set the parameter
+
+collect_rlzs = true
+
+in the `job.ini` file. That makes the risk part of an event based risk
+calculation even faster and more memory efficient than before, at the price of
+losing information about the specific realizations.
+
+For continental scale calculations setting `collect_rlzs = true` can
+make the difference between begin able to run a calculation and being
```
Comment (mmpagani) [inline doc/whats-new.md:125]: begin -> being

File: doc/whats-new.md [hunk 3]
```diff
@@ -1,362 +1,367 @@
...
+
+The GMPE library has been completely rewritten and the API for
+implementing new GMPEs has changed significantly. That means that if
+you have written a GMPE with the old API it will not work anymore once
+you upgrade to engine 3.12: you will have to rewrite it. This is quite
+simple is the GMPE is simple, but it can be quite difficult if the GMPE
+depends from other GMPEs in complex ways. Should you encounter any
+compatibility issue, please contact us.
+
+On the bright side, if are just using library GMPEs by calling the method
```
Comment (mmpagani) [inline doc/whats-new.md:188]: if -> if you

**Review by mmpagani — APPROVED** [review]
Thanks Miichele, I left a few minor comments.

File: doc/whats-new.md (hunk 1 above)
Comment (micheles) [reply doc/whats-new.md:63]: Yes, exclusively for risk, the option is ignored in hazard calculations.

File: doc/whats-new.md (hunk 3 above)
Comment (micheles) [reply doc/whats-new.md]: Done

File: doc/whats-new.md (hunk 2 above)
Comment (micheles) [reply doc/whats-new.md]: Done

**Review by raoanirudh — APPROVED** [review]

**Pull request merged**